### PR TITLE
Automated Tests

### PR DESCRIPTION
**Pull request by sohnryang**
We can now test the validity of all answers in one `pytest` command.

### CODE REVIEW

**Comment by sohnryang** [comment]
The CI is failing due to pytest failing back to default config. This is happening since no `*-answer.py` file is present in this branch, but we will add those files later so it is just a temporary issue.

File: .github/workflows/python-tests.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
```
Comment (hyojeongyunn) [inline .github/workflows/python-tests.yml]: Python version mismatches with name description. (..?)

Comment (sohnryang) [reply .github/workflows/python-tests.yml]: Fixed!

**Review by hyojeongyunn — APPROVED** [review]
👍

**Pull request merged**